content/projects/data-science-specific/data-visualisation/_index.md

```diff
@@ -5,22 +5,46 @@ ready: True
 
 ## Tutorials
 
-Go through the DataCamp course [Introduction to Data Visualisation with
-Python](https://www.datacamp.com/courses/introduction-to-data-visualization-with-python), or if you want to fast-track, complete Kaggle's [Data Visualisation: From Non-Coder to Coder](https://www.kaggle.com/learn/data-visualization-from-non-coder-to-coder)
+**Compulsory**: Go through the DataCamp course [Introduction to Data Visualisation with
+Python](https://www.datacamp.com/courses/introduction-to-data-visualization-with-python).
+
+**Optional**: Complete Kaggle's [Data Visualisation: From Non-Coder to Coder](https://www.kaggle.com/learn/data-visualization-from-non-coder-to-coder)
 
 
 ## Assignment 1
 Once you have completed the course, complete the DataCamp project
```

Comment:
If you want to be able to track these assignments separately (different deadlines, easy life for your code reviewers, etc) then they should be in separate _index.md pages. You can make some sub-directories in here. Look here for an example: https://github.com/Umuzi-org/tech-department/tree/master/content/projects/semitone-challenge